ci/vmtest/run_selftests.sh

```diff
@@ -52,11 +52,13 @@ test_progs_helper() {
   local selftest="test_progs${1}"
   local args="$2"
 
+  json_file="/${selftest}.json"
+
   foldable start ${selftest} "Testing ${selftest}"
   # "&& true" does not change the return code (it is not executed
   # if the Python script fails), but it prevents exiting on a
   # failure due to the "set -e".
-  ./${selftest} ${args} ${DENYLIST:+-d"$DENYLIST"} ${ALLOWLIST:+-a"$ALLOWLIST"} && true
+  ./${selftest} ${args} ${DENYLIST:+-d"$DENYLIST"} ${ALLOWLIST:+-a"$ALLOWLIST"} --json-summary "${json_file}" && true
```

Comment:
So if I understand correctly, we get a file with JSON encoded summary (in addition to the output that the selftest emits?) and then what? I don't see it being used anywhere? Would be useful to describe how that fits into a bigger picture in some commit or PR summary at the very least.

Reply:
> Would be useful to describe how that fits into a bigger picture in some commit or PR summary at the very least.

yeah, that's fair.

I will update the commit and PR.